### ISSUE TITLE
Investigate random errors

### ISSUE THREAD

**Issue by Southparkfan** [miraheze/mw-config#89]
I know this ticket is messy, but will clean it up asap.

.log:[fatal] [02a1c219] [no req]   ErrorException from line 264 of /srv/mediawiki/w/includes/exception/MWExceptionHandler.php: Fatal Error: Undefined constant 'MediaWiki\Extensions\OAuth\SpecialMWOAuthConsumerRegistration'
.log:[fatal] [031fe7fa] [no req]   ErrorException from line 264 of /srv/mediawiki/w/includes/exception/MWExceptionHandler.php: Fatal Error: Call to undefined function class_exist()
wikiolimpica.miraheze.org.log:[error] [72f74594] /wiki/Especial:Notifica%C3%A7%C3%B5es   ErrorException from line 78 of /srv/mediawiki/w/extensions/Echo/special/SpecialNotifications.php: PHP Notice: Undefined index: targetpage
meta.miraheze.org.log:[error] [c06f9d39] /w/index.php?title=Special:ListGroupRights&printable=yes   ErrorException from line 3454 of /srv/mediawiki/w/languages/Language.php: PHP Notice: Undefined index: 2

**Comment by Southparkfan** [comment]
The first two are human errors in eval.php scripts, so I don't need to look at them.
The last one is, however, not an error in eval.php:
```
[error] [f66e8744] /wiki/Special:ListGroupRights   ErrorException from line 3454
 of /srv/mediawiki/w/languages/Language.php: PHP Notice: Undefined index: 2
#0 /srv/mediawiki/w/languages/Language.php(3454): MWExceptionHandler::handleErro
r()
#1 /srv/mediawiki/w/includes/specials/SpecialListgrouprights.php(279): Language-
>listToText()
#2 /srv/mediawiki/w/includes/specials/SpecialListgrouprights.php(132): SpecialListGroupRights->formatPermissions()
#3 /srv/mediawiki/w/includes/specialpage/SpecialPage.php(384): SpecialListGroupRights->execute()
#4 /srv/mediawiki/w/includes/specialpage/SpecialPageFactory.php(582): SpecialPage->run()
#5 /srv/mediawiki/w/includes/MediaWiki.php(267): SpecialPageFactory::executePath()
#6 /srv/mediawiki/w/includes/MediaWiki.php(566): MediaWiki->performRequest()
#7 /srv/mediawiki/w/includes/MediaWiki.php(414): MediaWiki->main()
#8 /srv/mediawiki/w/index.php(41): MediaWiki->run()
#9 {main}
```

**Comment by Southparkfan** [comment]
Undefined index notice has been fixed in 7fdc4576d61caca7fac88709d7ac2f800ac8fbbb.

**Comment by Southparkfan** [comment]
Extra fixes have been implemented in 525805b500794e86b408351234d819b865fd7eb6, this issue is now finally fixed.
